package-lock.json

```diff
@@ -1,5739 +1,8 @@
 {
 	"name": "jquery.uls",
 	"version": "0.1.0",
-	"lockfileVersion": 2,
```

Comment:
This change is because of npm version 6. If we use npm 7, the lockversion will be 2 and diff will be minimal

Reply:
Thanks, fixed.